Assets/MixedRealityToolkit/_Core/Interfaces/DataProviders/SpatialObservers/IMixedRealitySurfaceObserver.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.DataProviders.SpatialObservers
+{
+    public interface IMixedRealitySurfaceObserver : IMixedRealitySpatialObserverDataProvider
```

Comment:
Please re-add SurfacePhysicsLayer property and the SurfacePhysicsLayerMask helper

Reply:
No.